src/PrestaShopBundle/Resources/config/services/bundle/repository.yml

```diff
@@ -148,3 +138,8 @@ services:
     factory: [ '@doctrine.orm.default_entity_manager', getRepository ]
     arguments:
       - PrestaShopBundle\Entity\AuthorizedApplication
+    public: true
+
+  prestashop.core.admin.feature_flag.repository:
```

Comment:
The alias has been moved e should keep it under the aliased service

Reply:
fixed